src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1224,7 +1208,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
             return this.DEVELOPER_PREVIEW_WATERMARK
         }
 
-        // The watermark applies when any tracking issue for the sections of document is not in status DONE
+        // The watermark applies when any tracking issue for the sections of document is not in status DONE and is to be generated in the environment
```

Comment:
Can you fix the comment? It still includes a previous assumption based on the status of the tracking issue. Now, it refers to the statii of all document chapters.

Reply:
Done